lib/common/utils.ts

```diff
@@ -569,3 +569,15 @@ export function findEventTask(target: any, evtName: string): Task[] {
   }
   return result;
 }
+
+Zone[zoneSymbol('patchEventTargetMethods')] = patchEventTargetMethods;
```

Comment:
I think we should expose the minimal set. I will remove the rest of the exports and just keep `patchEventTargetMethods `